manager/controllers/app/m4dapplication_controller.go

```diff
@@ -283,20 +277,24 @@ func (r *M4DApplicationReconciler) ConstructDataInfo(datasetID string, req *modu
 }
 
 // NewM4DApplicationReconciler creates a new reconciler for Blueprint resources
```

Comment:
blueprint or plotter

Reply:
It is neither, actually. m4dapplication reconciler reconciles m4dapplications.